core/src/test/scala/integration/kafka/api/BaseProducerSendTest.scala

```diff
@@ -175,6 +175,29 @@ abstract class BaseProducerSendTest extends KafkaServerTestHarness {
     sendAndVerifyTimestamp(producer, TimestampType.CREATE_TIME)
   }
 
+  protected def sendAndVerify(producer: KafkaProducer[Array[Byte], Array[Byte]], numRecords: Int = numRecords) {
+    val partition = new Integer(0)
+    try {
+      TestUtils.createTopic(zkUtils, topic, 1, 2, servers)
+
+      val recordAndFutures = for (i <- 1 to numRecords) yield {
+        val record = new ProducerRecord(topic, partition, "key".getBytes, "value".getBytes)
```

Comment:
Would it not be a little better to include `i` in the key and value from a debugging perspective?